scripts/itkdev-docker-compose

```diff
@@ -32,6 +32,8 @@ Commands:
   sql:port:
                 Display the exposed MariaDB SQL server port.
 
+  open:url:
```

Comment:
This should be `mailhog:url` and so on …

Reply:
Fixed